src/bluesearch/entrypoint/database/download.py

```diff
@@ -163,7 +165,68 @@ def run(source: str, from_month: datetime, output_dir: Path, dry_run: bool) -> i
         logger.info(f"Start downloading {source} papers.")
         download_s3_articles(bucket, url_dict, output_dir)
         return 0
+    elif source == "arxiv":
+        logger.info("Loading libraries")
+        from google.cloud.storage import Client
+
+        from bluesearch.database.download import download_gcs_blob, get_gcs_urls
+
+        client = Client.create_anonymous_client()
+        bucket = client.bucket("arxiv-dataset")
+        if from_month < datetime(2007, 4, 1):
+            logger.error(
+                "The papers from before April 2007 follow a different format "
+                "and can't be downloaded. Please contact the developers if you "
+                "need them. To proceed please re-run the command with a "
+                "different starting month."
+            )
```

Comment:
This is nice. However, I feel like we should also do the same thing with other sources since basically all of them changed their "storing logic" at one point in the past. Might be worth creating a separate PR for this.

Reply:
Yeah, that's also one of the points I put into "to discuss" in the description. I think the problem here is not the downloading itself, which is easy, but the fact that we'd download `q-bio` only (or another category) since that's how they used to sort them. I wasn't sure how that would interect with our filtering.